integration/tests/basic/rpc/test_rpc_get_transaction.py

```diff
@@ -231,7 +257,11 @@ def test_get_transaction_receipt(self, method, json_rpc_client):
         assert result["to"].upper() == tx_receipt["to"].upper()
         assert result["contractAddress"] is None
         assert result["logs"] == []
+        if method == "eth_getTransactionReceipt":
+            EthGetTransactionReceiptResult(**response)
+        
 
+    # Geth returns code 32601
```

Comment:
Is the issue created?

Reply:
Geth returns error 32601 because it does not support neon_* operations. I think it's ok just remove a comment in this case because this test is `neon_only`.